packages/core-data/src/entities.js

```diff
@@ -205,7 +205,7 @@ export const prePersistPostType = ( persistedRecord, edits ) => {
  * @return {Promise} Entities promise
  */
 async function loadPostTypeEntities() {
```

Comment:
FYI these are the changes from https://github.com/WordPress/gutenberg/pull/37685